### ISSUE TITLE
Transformation error: crashes on TSInstantiationExpression

### ISSUE THREAD

**Issue by Ayc0** [eps1lon/types-react-codemod#91]
When I run the code on one of my codebases, it crashed with the error:

```
Transformation error (did not recognize object of type "TSInstantiationExpression")
Error: did not recognize object of type "TSInstantiationExpression"
```

I think it's related to a new TS feature like `Map<string, Error>`:

```js
function makeBox<T>(value: T) {
    return { value };
};

const makeStringBox = makeBox<string>; // <- This is a "TSInstantiationExpression"
const stringBox = makeStringBox('abc');

const ErrorMap = Map<string, Error>; // <- This is a "TSInstantiationExpression"
const errorMap = new ErrorMap();
```

See the following code on astexplorer.net ([view snippet](https://astexplorer.net/#/gist/4a25278ef52f5d34371320905a5fcd35/latest)) with JS and recast (jscodeshift's engine):
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/22725671/182427516-a8bfe6ec-aedf-4bf3-aab5-f88c4dd07adf.png">

**Comment by eps1lon** [comment]
Should be fixed in 1.3.0.

**Comment by Ayc0** [comment]
Thanks!
